manifests/ols.yaml

```diff
@@ -45,7 +45,5 @@ spec:
         ports:
         - containerPort: 8443
           name: https
-        securityContext:
-          allowPrivilegeEscalation: false
       - name: ols
-        image: ${OLS_IMAGE}
\ No newline at end of file
+        image: quay.io/openshift/lightspeed-service-api:latest
```

Comment:
i'm not clear on your motivation to drop the use of env vars in these files.  You indicate it's preparatory work for creating the helm chart but i'm not sure why you can't make this change when you introduce the helm chart structure.

If you are determined to drop these env vars now (i don't overly care either way since i doubt anyone is really using this much today), then the readme should also be updated since it mentions using `envsubst` to deploy these resources, which has no purpose if there are no env vars to be substituted.

Reply:
Yes, I agree that it will be necessary to update Readme.
The motivation was simple - to start the service, I had to either create additional environment variables and use Kustomize or modify manifests directly.

I have the following idea -

- make working manifests so that they can be used without Helm (if someone really want to)
- make Helm and there we can configure the service as user wants

Reply:
What to do in the end in this PR?
Leave variables as, Kustomize and remove SCC?

Reply:
> What to do in the end in this PR?
> Leave variables as is, Kustomize and remove SCC?

yes i think that is correct (assuming that my updates to our image have fixed the issue you were hitting).